apps/admin-x-activitypub/src/components/layout/Sidebar/Sidebar.tsx

```
@@ -36,6 +39,22 @@ const Sidebar: React.FC = () => {
+                        {allFlags.map((flag) => {
+                            if (flags[flag]) {
+                                return (
+                                    <div key={flag} className="flex items-center justify-between gap-1">
+                                        <span className="font-mono text-xs">{flag}</span>
+                                        <span className='text-green-800 inline-flex items-center rounded bg-green-100 px-1 py-0.5 text-xs font-medium'>
+                                            ON
+                                        </span>
+                                    </div>
+                                );
+                            }
+                            return (<></>);
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)